pymilo/chains/clustering_chain.py

```diff
@@ -5,18 +5,23 @@
 from ..transporters.general_data_structure_transporter import GeneralDataStructureTransporter
 from ..transporters.function_transporter import FunctionTransporter
 
-from ..pymilo_param import SKLEARN_CLUSTERING_TABLE
-
+from ..pymilo_param import SKLEARN_CLUSTERING_TABLE, NOT_SUPPORTED
 from ..exceptions.serialize_exception import PymiloSerializationException, SerilaizatoinErrorTypes
 from ..exceptions.deserialize_exception import PymiloDeserializationException, DeSerilaizatoinErrorTypes
 from traceback import format_exc
 
-
+bisecting_kmeans_support = True if SKLEARN_CLUSTERING_TABLE["BisectingKMeans"] != NOT_SUPPORTED else False
```

Comment:
`bisecting_kmeans_support = not (SKLEARN_CLUSTERING_TABLE["BisectingKMeans"] != NOT_SUPPORTED)`

Reply:
Actually it is `bisecting_kmeans_support = SKLEARN_CLUSTERING_TABLE["BisectingKMeans"] != NOT_SUPPORTED`
but I got your point and I will apply them all.

Reply:
applied.